agent/agentcontainers/api.go

```diff
@@ -129,7 +130,7 @@ func WithDevcontainerCLI(dccli DevcontainerCLI) Option {
 }
 
 // WithSubAgentClient sets the SubAgentClient implementation to use.
-// This is used to list, create and delete Dev Container agents.
+// This is used to list, create and delete devcontainer agents.
```

Comment:
while we're nitpicking: eats, shoots, and leaves ;-) 

```suggestion
// This is used to list, create, and delete devcontainer agents.
```